versioned_docs/version-0.10/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,30 @@ sidebar_label: Release Notes
 id: releases
```

Comment:
@maroshii these changes should go on https://github.com/okteto/docs/blob/main/src/content/enterprise/install/releases.mdx